Practice/Practice.Repository/HumanResourcesRepository.cs

```diff
@@ -31,5 +33,70 @@ public async Task<List<ViewModels.Employee>> GetEmployees()
 
             return employees;
         }
+
+        public async Task<ViewModels.Employee> GetEmployeeById(int id)
+        {
+            if (id == 0)
+                return null;
+
+            var employee = await context.vEmployee.FirstOrDefaultAsync(e => e.BusinessEntityID == id);
```

Comment:
SingleAsync - jeśli pobierasz po kluczu głównym to zawsze powinieneś się spodziewać zwrócenia pojedynczego wyniku